src/lsrs/web/utility/middleware.ts

```diff
@@ -112,3 +114,19 @@ export const formidableUploadMiddleware = async (request: Request, response: Res
 
     next();
 }
+
+/**
+ * Creates a request tracking middleware for Express. Allows TSLog logging library to include a shared request ID
+ * across the log messages created within a single web request.
+ *
+ * @param request Express Request object
+ * @param response Express Response object
+ * @param next Express next function
+ */
+export const requestTrackingMiddleware = async (request: Request, response: Response, next: NextFunction) => {
```

Comment:
Nice!

Reply:
Thanks! :D This is the recommended way to implement request tracking with TSLog. :)